percona/watcher/wal.go

```diff
@@ -7,16 +7,15 @@ import (
 	"time"
 
 	"github.com/pkg/errors"
-	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/fields"
-	"k8s.io/apimachinery/pkg/labels"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/event"
 
 	"github.com/percona/percona-postgresql-operator/internal/logging"
 	"github.com/percona/percona-postgresql-operator/percona/clientcmd"
 	"github.com/percona/percona-postgresql-operator/percona/pgbackrest"
+	common "github.com/percona/percona-postgresql-operator/percona/postgres"
```

Comment:
i don't like this `common`, why it was necessary?

Reply:
So we need to use both "github.com/percona/percona-postgresql-operator/internal/postgres"
and "github.com/percona/percona-postgresql-operator/percona/postgres".
To keep both packages and avoid using variable, I can suggest to rename postgres package in percona directory to perconapostgres. What do you think? 
@egegunes

Reply:
In that case, we generally use `util` rather than `common`, maybe you could have alias here like `pgUtil` or something like that. Or maybe even `perconaPostgres` (probably too long) or maybe `perconaPG`?

Reply:
If we want use pgUtil it might be better to this code to utils? 
perconaPG I like the most.

Reply:
Yeah, I believe you can go with perconaPG. Besides this, this PR looks good to me.